constraints.txt

```diff
@@ -26,6 +26,7 @@ distro==1.6.0
 enrich==1.2.6
 execnet==1.9.0
 idna==3.2
+importlib-metadata==4.8.1 ; python_version > "3.7"
```

Comment:
```suggestion
importlib-metadata==2.1.1 ; python_version <= "3.7"
importlib-metadata==4.8.1 ; python_version > "3.7"
```

Reply:
This file is not editable by hand, the change that you proposed will break the linting as pip-compile will remove your change.